kimchi/src/circuits/polynomials/foreign_mul/mod.rs

```diff
@@ -0,0 +1,104 @@
+//! Foreign field multiplication gate
+
+/// <https://hackmd.io/XZUHHGpDQsSOs0dUGugB5w>
+///
+/// Globals:
+///     * n: native field modulus
+///     * p: foreign field modulus
+///
+/// Inputs:
+///     * a: left foreign field element operand a \in Fp
```

Comment:
`a` isn't really 'in `Fp`', right? It's probably more correct to say 'as Fp limbs', this feels confusing as currently written.

Reply:
Well Fp is the foreign field and Fn is native (see above).

Reply:
Fixed